src/lib/y2storage/blk_device.rb

```diff
@@ -377,7 +377,9 @@ def mount_point
     # @return [LvmPv] nil if neither the raw device or its encrypted version
     #   are used as physical volume
     def lvm_pv
-      descendants.detect { |dev| dev.is?(:lvm_pv) && dev.plain_blk_device == plain_device }
+      descendants.detect do |dev|
```

Comment:
np: I prefer `#find` to `#detect` (just like mentioned in [this guide](https://github.com/rubocop-hq/ruby-style-guide#mapfindselectreduceincludesize)).